assets/js/components/setup/search-console.js

```diff
@@ -181,11 +146,6 @@ class SearchConsole extends Component {
 			try {
 				await this.insertPropertyToSearchConsole( siteURL );
 
-				this.setState( {
-					loading: false,
-					connected: true,
-				} );
```

Comment:
Why did you remove this?

Reply:
@felixarntz because state is already set in `insertPropertyToSearchConsole()` https://github.com/google/site-kit-wp/blob/010cfb9e1e01d3aab790e6dfb9656e05a26fefd2/assets/js/components/setup/search-console.js#L129-L136